.github/workflows/codeql-analysis.yml

```diff
@@ -31,11 +31,6 @@ jobs:
         # a pull request then we can checkout the head.
         fetch-depth: 2
 
-    # If this run was triggered by a pull request event, then checkout
```

Comment:
Since I was dealing with actions already I also updated `codeql-analysis.yml` according to this warning:

![image](https://user-images.githubusercontent.com/47985/118586485-06798680-b7c5-11eb-8fb0-49c46f8bac87.png)